zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/tenant/AddEntityTenantTest.java

```diff
@@ -27,19 +30,17 @@ public class AddEntityTenantTest {
       new RecordingExporterTestWatcher();
 
   @Test
-  public void shouldAddEntityToTenant() {
-    // given
-    final var userKey =
-        engine
-            .user()
-            .newUser("username")
-            .withName("Foo Bar")
-            .withEmail("foo@bar.com")
-            .withPassword("password")
-            .create()
-            .getValue()
-            .getUserKey();
+  public void shouldAddEntitiesToTenant() {
+    // We can't use JUnit 5 parameterized tests due to JUnit 4's @Rule incompatibility, and JUnit
+    // 4's parameterized tests are not suitable for this test class    shouldAddEntityToTenant(USER,
+    //  createUser());
+    shouldAddEntityToTenant(GROUP, createGroup());
+    shouldAddEntityToTenant(MAPPING, createMapping());
+    shouldAddEntityToTenant(USER, createUser());
+  }
```

Comment:
🔧  We can also split this test into a separate JUnit 4 parameterized test class or split these 3 cases into 3 separate test methods here. This way we don't have any test isolation.